packages/web-api/src/types/request/team.ts

```diff
@@ -5,25 +5,27 @@ import {
   CursorPaginationEnabled,
   TraditionalPagingEnabled,
 } from './common';
+import { OptionalArgument } from '../helpers';
 
 // https://api.slack.com/methods/team.accessLogs
-export interface TeamAccessLogsArguments extends TokenOverridable, CursorPaginationEnabled,
-  TraditionalPagingEnabled, OptionalTeamAssignable {
+export type TeamAccessLogsArguments = OptionalArgument<TokenOverridable & CursorPaginationEnabled &
+TraditionalPagingEnabled & OptionalTeamAssignable & {
   /**
    * @description End of time range of logs to include in results (inclusive) as a UNIX timestamp in seconds.
    * Default to now.
    */
   before?: number;
-}
+}>;
 // https://api.slack.com/methods/team.billableInfo
-export interface TeamBillableInfoArguments extends TokenOverridable, CursorPaginationEnabled, OptionalTeamAssignable {
+export type TeamBillableInfoArguments = OptionalArgument<TokenOverridable & CursorPaginationEnabled &
+OptionalTeamAssignable & {
   /** @description A user to retrieve the billable information for. Defaults to all users. */
   user?: string;
-}
+}>;
 // https://api.slack.com/methods/team.billing.info
-export interface TeamBillingInfoArguments extends TokenOverridable {}
+export type TeamBillingInfoArguments = OptionalArgument<TokenOverridable>;
 // https://api.slack.com/methods/team.info
-export interface TeamInfoArguments extends TokenOverridable {
+export type TeamInfoArguments = OptionalArgument<TokenOverridable & {
   /**
```

Comment:
☝️ This one has no type tests, but it's optional too

Reply:
Can we add missing type tests while we're here?

Reply:
Sure!

Reply:
Done in f98f005fb4a0e0fc72c3cf82b64b3cbc6b1a919b